src/ansiblelint/schemas/changelog.json

```
@@ -18,45 +46,182 @@
+                "type": "string"
+              },
+              "type": "array"
+            },
             "minor_changes": {
               "items": {
                 "type": "string"
               },
               "type": "array"
             },
             "release_summary": {
+              "markdownDescription": "This must be valid [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText).",
+              "title": "Release Summary",
               "type": "string"
+            },
+            "removed_features": {
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
+            },
+            "security_fixes": {
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
+            },
+            "trivial": {
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
             }
           },
           "type": "object"
         },
         "codename": {
           "type": "string"
         },
+        "fragments": {
+          "items": {
+            "type": "string"
+          },
+          "markdownDescription": "List of strings representing filenames with framents.",
+          "type": "array"
+        },
         "modules": {
           "items": {
             "properties": {
               "description": {
-                "markdownDescription": "Value of `short_description from plugin `DOCUMENTATION`.",
+                "markdownDescription": "Value of `short_description` from plugin `DOCUMENTATION`.",
                 "title": "Description",
                 "type": "string"
               },
               "name": {
-                "markdownDescription": " It must not be the FQCN, but the name inside the collection.",
+                "markdownDescription": "It must not be the FQCN, but the name inside the collection.",
                 "pattern": "[a-zA-Z0-9_]+",
                 "title": "Short module name",
                 "type": "string"
               },
               "namespace": {
-                "markdownDescription": "Must be `null` for plugin and objects. For modules it must be either empty string. The namespace is used to group new modules by their namespace inside the collection.",
+                "markdownDescription": "Must be `''` for modules directly in `plugins/modules/`, or the dot-separated list of directories the module is in inside the `plugins/modules/` directory. This is mostly relevant for large collections such as `community.general` and `community.network`. For example, the `community.general.docker_container` module is in the directory `plugins/modules/cloud/docker/`, hence its namespace must be `cloud.docker`. The namespace is used to group new modules by their namespace inside the collection.",
                 "title": "Namespace",
-                "type": ["string", "null"]
+                "type": ["string"]
```

Shouldn't this be
```suggestion
                "type": "string"
```
? (I guess both work, but this is the syntax you use in other cases.)